### DISCUSSION TITLE
Write part of equation aligned to right

### DISCUSSION THREAD

**Discussion by JosephBoom02** [typst/typst#3042] [Q&A] [answered]
How can I write part of equation aligned to right (the j=1,...,n in the image) on typst, like with the \tag* command in Latex?
![Screenshot from 2023-12-21 23-48-37](https://github.com/typst/typst/assets/30532436/2084a53d-4e2c-4b1c-8976-c2568bf33769)

**Comment by xkevio** [comment, marked as answer]
Easiest way for automatic layouting would be a `grid`, like so:
```typ
#grid(columns: (1fr, 1fr, 1fr), column-gutter: 1fr,
  [],
  $ x_(ell, j) = sum_(i=1)^r sum_(q = 1)^h_i gamma_(j i q) t_(q-1)e^(lambda_i t) $,
  align(horizon)[$ j = 1,...,n $]
)
```
If you need it often, you can wrap that in your own `tag`-function.

You can also manually adjust with `#h()` but that's of course not ideal.

**Reply by JosephBoom02** [reply]
Thank you so much
